plat/kvm/Config.uk

```diff
@@ -101,6 +101,8 @@ config KVM_VMM_FIRECRACKER
 	bool "Firecracker"
 	select KVM_BOOT_PROTO_LXBOOT
 	select HAVE_MMIO
+	select HAVE_PS2
```

Comment:
I will make it so that the keyboard driver selects it isntead, as  if announcing "hey, enabling me registers shutdown handler"

Reply:
Sounds good. Besides, the result is the same: shutdown is still enabled but explicitly. One thing is that in that case we assume that Unikraft will always be able to shutdown, as long as a method to initiate shutdown is provided to the user. If the platform does not support it, then it will probably ignore the event, but Unikraft will go all the way to halt nevertheless. One thing we could improve then is renaming `HAVE_SHUTDOWN` to something that indicates more clearly that we are referring to having a shutdown method, to avoid confusion.

Also if possible, please update the commit message with a brief description of the design.